backend/src/game/dice_handler.py

```diff
@@ -0,0 +1,26 @@
+import backend.src.game.dice as dice
+import backend.src.game.constants as constants
+
+
+class DiceHandler:
+    dice_values = []
+    re_rolls_left = constants.DEFAULT_RE_ROLL_COUNT
+
+    def start_turn(self):
+        self.dice_values = dice.roll_many(constants.DEFAULT_DICE_COUNT)
+        self.re_rolls_left = constants.DEFAULT_RE_ROLL_COUNT
+
+    def re_roll_dice(self, indexes_of_dice_to_re_roll):
+        if self.re_rolls_left <= 0:
+            return
+
+        if type(indexes_of_dice_to_re_roll) == int:
+            self.dice_values[indexes_of_dice_to_re_roll] = dice.roll()
```

Comment:
How is there a minus re_reoll in line 23 but not here, I think this function could be refactored

Reply:
Ah you're right. That was a big miss. This was already refactored anyways.